app/controllers/item.js

```diff
@@ -31,134 +32,141 @@ var UserSourceAuth = require('app/models/userSourceAuth');
 var UserStorageAuth = require('app/models/userStorageAuth');
 var validateParams = require('app/lib/validateParams');
 
+var util= require('util');
+
 var queue = kue.createQueue();
 
-queue.process('storeItemData', function(queueJob, done) {
-  debug('process queueJob %s', queueJob.id);
+var tools = require('app/lib/utils/debuggingTools');
 
-  var getItem = (done) => {
-    Item.findById(queueJob.data.itemId, (error, item) => {
-      if (error) {
-        done(error);
-      } else if (!item) {
-        done(new Error('Item with queueJob.data.itemId not found'));
-      } else {
-        done(undefined, item);
-      }
-    });
-  };
+const STORE_ITEM_DATA = 'storeItemData';
+
+queue.process(STORE_ITEM_DATA, function(queueJob, done) {
+    console.log('process queueJob %s', queueJob.id);
```

Comment:
Please replace four-space indentation with two-space throughout all your changes since that's the standard throughout the codebase.